### ISSUE TITLE
Massive Breakouts in random directions.

### ISSUE THREAD

**Issue by AnthonyJQQ** [QuadQuestions/qq190rtf_tune#1]
@hollydiego 

# hollydiego [6:54 AM] 
I have two builds with the BetaflightF3 boards. Under 3.1.7. and only on full throttle I have massive break outs in random directions. This is a huge problem especially when you do power loops... the quad will crash immediately. Now testing the new QQ beta settings under 3.1.7 the same effect! I really tried all settings spending hours but always the same effect on full throttle. Going back to 3.1.6. or lower and on both the BetaflightF3 board and QQ everything is fine. I will contact Boris to discuss this strange behaviour. I have seen comments on the net bout BF 3.1.7 talking about same or similar effects. Cheers

**Comment by AnthonyJQQ** [comment]
Hi Holly, this is a pretty known occurrence in 3.1.7.  Can you tell me if you have tried these tunes on the RTF, or is your hardware strictly limited to the betaflight fc? 

Thanks!

Anthony

**Comment by hollydiego** [comment]
Yes I also tried it on my two RTF QQ190. Same issue.

**Comment by AnthonyJQQ** [comment]
@hollydiego what ESC firmware are you running on the RTF?  Should be BLHeli 16.5.  Use these INI files:
[QQ190 BLHeli_TBS 2in1 - F_H_30 - Rev. 16.5 - Multi_161223.ixi.zip](https://github.com/QuadQuestions/qq190rtf_tune/files/1006612/QQ190.BLHeli_TBS.2in1.-.F_H_30.-.Rev.16.5.-.Multi_161223.ixi.zip)

**Comment by hollydiego** [comment]
I used 16.4, 16.5 and 16.6. Makes no difference.

**Comment by terrorswellfpv** [comment]
@hollydiego  Interesting! I am running bf fc with 3.1.7 with no issues. What is the esc/motor combo?

**Comment by hollydiego** [comment]
@terrorswellfpv I use Brotherhobby T1 motors and Aikon 30A V3 ESCs on the BF FC. I find it interesting that we have these issues on four different builds.

**Comment by AnthonyJQQ** [comment]
Thanks for the comments, I would like to keep the results shared here RTF centric, but I agree that there appears to be an issue with BF3.1.7 at this point-

**Comment by hollydiego** [comment]
@AnthonyJQQ Agree... and I really tried everything you can do on the RTF QQ190. Going to a lower BF version and no problem at all.
